libs/opensaml/build.gradle

```diff
@@ -69,6 +69,7 @@ tasks.shadowJar {
     exclude 'org/publicsuffix/**'
     exclude 'org/slf4j/**'
     exclude 'javax/**'
+    exclude 'META-INF/maven/commons-io/commons-io/**'
```

Comment:
`org.apache.commons:commons-lang3 (opensaml-3.2.0.0-all.jar)`

Should we also exclude these files?

```
META-INF/maven/org.apache.commons/
META-INF/maven/org.apache.commons/commons-lang3/
META-INF/maven/org.apache.commons/commons-lang3/pom.xml
META-INF/maven/org.apache.commons/commons-lang3/pom.properties
```